specification/gsm/resource-manager/Microsoft.ManagementPartner/preview/2018-02-01/gsm.json

```diff
@@ -0,0 +1,412 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ACE Provisioning GSM API",
+    "description": "This API describe ACE Provisioning GSM",
+    "version": "2018-02-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.ManagementPartner/partners/{partnerId}": {
+      "get": {
+        "summary": "Get a specific `Partner`.",
+        "description": "Get the management partner using the partnerId, objectId and tenantId.",
+        "operationId": "Partner_Get",
+        "x-ms-examples": {
+          "GetPartnerDetails": {
+            "$ref": "./examples/GetPartnerDetails.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/PartnerIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "tags": [
+          "partner"
+        ],
+        "responses": {
+          "200": {
+            "description": "Get the details of the `Partner`.",
+            "schema": {
+              "$ref": "#/definitions/PartnerResponse"
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "put": {
+        "summary": "Create a specific `Partner`.",
+        "description": "Create a management partner for the objectId and tenantId.",
+        "operationId": "Partner_Put",
```

Comment:
The name should be "Create" or "CreateOrUpdate", in 1st case "Patch" should also be named "Update"

Reply:
Sorry, what should be changed to "Create", 
Partner_Put should be changed to Parnter_Create?

Reply:
Yes - sorry for the confusion

Reply:
thanks, will update